packages/flutter_tools/test/integration.shard/isolated/native_assets_without_cbuild_assemble_test.dart

```diff
@@ -138,8 +138,11 @@ void main(List<String> args) async {
           mode: ProcessStartMode.inheritStdio,
         );
         expect(await process.exitCode, 0);
-      });
+      },);
```

Comment:
Are you preparing for when the formatter is going to run here? ;)

Reply:
In my head the formatter is already on.